Demo/build.gradle

```diff
@@ -62,6 +63,11 @@ dependencies {
     implementation deps.composeRuntimeLivedata
     implementation deps.composeFoundation
     implementation deps.composeMaterial
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
Is the legacy-support library needed? Not sure if this was added for the Navigation Component.

Reply:
Yeah not sure how this snuck in there - removed it.